src/index.jsx

```diff
@@ -52,7 +52,7 @@ subscribe(APP_READY, () => {
           <Route path={ROUTES.Detail.EDIT} element={<LibraryEditPage />} />
           <Route path={ROUTES.Detail.ACCESS} element={<LibraryAccessPage />} />
           <Route path={ROUTES.Detail.IMPORT} element={<CourseImportPage />} />
-          <Route path={`${ROUTES.Block.HOME}/*`} element={<LibraryBlockPage />} />
+          <Route path={`${ROUTES.Block.HOME}`} element={<LibraryBlockPage />} />
```

Comment:
```suggestion
          <Route path={ROUTES.Block.HOME} element={<LibraryBlockPage />} />
```

Reply:
Fair. Going to wait for Dave's test then I'll put up an amended version.